_docs_v5/2_configuration/target.md

```diff
@@ -4,8 +4,8 @@ title: Target Configuration
 toc: true
 ---
 
-Each target is self contained in a target directory named after the target. In the root of the target directory there is a configuration file named target.txt which configures the individual target.
+Targets are the external embedded systems that COSMOS connects to. Targets are defined by the top level [TARGET]({{site.baseurl}}/docs/v5/plugins#target-1) keyword in the plugin.txt file. Each target is self contained in a target directory named after the target. In the root of the target directory there is a configuration file named target.txt which configures the individual target.
```

Comment:
Added this line to make it clear that the TARGET keyword is defined over in the Plugins page. This is for the target.txt keywords.